docs/sources/setup/migrate/ssd-to-distributed/_index.md

```diff
@@ -0,0 +1,372 @@
+    ```
+   {{< admonition type="caution" >}}
+   It is important to allow all components to fully spin up before proceeding to the next stage. You can check the status of the components using the following command:
+
+    ```bash
+    kubectl get pods -n loki
+    ```
+   Let all components reach the `Running` state before proceeding to the next stage.
+   {{< /admonition >}}
+
+## Stage 2: Transitioning to Distributed components
+
+The final stage of the migration involves transitioning all traffic to the distributed components. This is done by scaling down the SSD components and swapping the `deploymentMode` to `Distributed`. To do this:
+
+1. Create a copy of `values-migration.yaml` and name it `values-distributed.yaml` (Once again this can all be done in the original file).
+
+    ```bash
+    cp values-migration.yaml values-distributed.yaml
+    ```
+2. Next modify the following parameters; `deploymentMode` and components based on the annotations below.
+   ```yaml
+   ---
+   loki:
+     schemaConfig:
+       configs:
+         - from: "2024-04-01"
+           store: tsdb
+           object_store: s3
+           schema: v13
+           index:
+             prefix: loki_index_
+             period: 24h
+     storage_config:
+       aws:
+         region: eu-central-1
+         bucketnames: aws-chunks-bucket
+         s3forcepathstyle: false
+     ingester:
+       chunk_encoding: snappy
+       wal:
+         flush_on_shutdown: true
+     ruler:
+       enable_api: true
+       storage:
+         type: s3
+         s3:
+           region: eu-central-1
+           bucketnames: aws-ruler-bucket
+           s3forcepathstyle: false
+         alertmanager_url: http://prom:9093
+     querier:
+       max_concurrent: 4
+
+     storage:
+       type: s3
+       bucketNames:
+         chunks: "aws-chunks-bucket"
+         ruler: "aws-ruler-bucket"
+       s3:
+         region: eu-central-1
+
+   # Important: Make sure to change this to Distributed
+   deploymentMode: Distributed
+
+   # SSD
+   # Scale down the SSD components
+   backend:
+     replicas: 0
+   read:
+     replicas: 0
+   write:
+     replicas: 0
+
+   # Distributed Loki
+   ingester:
+     replicas: 3
+     zoneAwareReplication:
+       enabled: false
+   querier:
+     replicas: 3
+     maxUnavailable: 0
+   queryFrontend:
+     replicas: 2
+     maxUnavailable: 0
+   queryScheduler:
+     replicas: 2
+   distributor:
+     replicas: 2
+     maxUnavailable: 0
+   compactor:
+     replicas: 1
+   indexGateway:
+     replicas: 2
+     maxUnavailable: 0
+   ruler:
+     replicas: 1
+     maxUnavailable: 0
+
+   # Single binary Loki
+   singleBinary:
+     replicas: 0
+
+   minio:
+     enabled: false
+   ```
+  Here is a breakdown of the changes:
+  * `deploymentMode: Distributed`: This will allow for the distributed components to run in isolation.
+  * Scale down all SSD components to `0`.
+
+
+## Migrating from SSD to Single Binary
```

Comment:
I wouldn't expect to find this in a topic where the heading is "Migrate from SSD to distributed."  I think we can delete this section.